### DISCUSSION TITLE
nrf52840 DK possible ?

### DISCUSSION THREAD

**Discussion by DusJson** [Samsung/ioter#52] [Q&A] [answered]
Hi,
Does Nordic nrd52840 DK board also support ?
Can I get the guide ?

**Comment by sangsoolee** [comment]
@jinwon0325kim @ejunu 
Can you check and help it ?

**Comment by ejunu** [comment, marked as answer]
I checked the ioter run on nrf52840 dk board. Follow the link below to build RCP and run the ioter.
 
[How to build rcp on nrf52840 DK board](https://openthread.io/codelabs/openthread-hardware#0)
